drivers/test.go

```diff
@@ -8,27 +8,29 @@ import (
 	"testing"
 
 	"github.com/filecoin-project/go-address"
+	"github.com/ipfs/go-cid"
+	datastore "github.com/ipfs/go-datastore"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
 	abi_spec "github.com/filecoin-project/specs-actors/actors/abi"
 	big_spec "github.com/filecoin-project/specs-actors/actors/abi/big"
 	builtin_spec "github.com/filecoin-project/specs-actors/actors/builtin"
 	account_spec "github.com/filecoin-project/specs-actors/actors/builtin/account"
 	cron_spec "github.com/filecoin-project/specs-actors/actors/builtin/cron"
 	init_spec "github.com/filecoin-project/specs-actors/actors/builtin/init"
 	market_spec "github.com/filecoin-project/specs-actors/actors/builtin/market"
-	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	miner_spec "github.com/filecoin-project/specs-actors/actors/builtin/miner"
```

Comment:
changed for consistency

Reply:
As a general comment: most of the imports in this project are from specs-actors. Name collisions don't look too difficult to avoid, `miner` was the most likely, if anything (and `init_` to avoid the Go reserved word). This convention will be difficult to enforce if many people contribute to this project. It won't last.

I suggest changing the convention so that the easy, tool-supported thing is the default (in a different PR).